src/components/SplitScreen.module.css

```diff
@@ -46,10 +46,11 @@
     }
     .slidingPane {
       background-color: rgba(255, 255, 255, 0.9);
-      bottom: -70%;
```

Comment:
curious about this change:

I might be causing this problem

![](http://g.recordit.co/IDG0i9ADnI.gif)